jmix-flowui/flowui/src/main/resources/io/jmix/flowui/module.properties

```diff
@@ -14,4 +14,6 @@
 # limitations under the License.
 #
 
-jmix.flowui.menu-config = io/jmix/flowui/menu.xml
\ No newline at end of file
+jmix.flowui.menu-config = io/jmix/flowui/menu.xml
+
+jmix.core.exclude-beans=contextInitializer
```

Comment:
Let's add a comment that explains why we add this

Reply:
Fixed